src/internal/components/ValidationAnimationContainer.tsx

```diff
@@ -19,6 +19,9 @@ const fadeIn = keyframes`
 // using easeOutQuint easing fn https://easings.net/#easeOutQuint
 const AnimatedElement = styled.div<Props>`
   animation: ${props => props.show && css`170ms ${fadeIn} cubic-bezier(0.44, 0.74, 0.36, 1);`};
+  @media (prefers-reduced-motion) {
```

Comment:
This was causing a bit of flakiness when it came to tests, as Axe would run before the animation is complete. Since the animation adjusts the opacity of the element, it would throw a color-contrast violation if scanned before the animation completed. 

This addition allows the animation to be removed when `prefers-reduced-motion` is true.